### PR TITLE
Fix the dangling symbolic link in git hooks

### PR DESCRIPTION
**Pull request by liuxintong**
Just as `ln --help` said, the 1st param `TARGET` should be a relative path compared to the 2nd param `LINK_NAME`.

> Symbolic links can hold arbitrary text; if later resolved, a relative link is interpreted in relation to its parent directory.

### CODE REVIEW

File: contributors/workflow.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ building, doc generation, etc:
 
 ```sh
 cd $working_dir/tidb/
-ln -s hooks/pre-commit .git/hooks/
+ln -s ../../hooks/pre-commit .git/hooks/
```
Comment (zz-jason) [inline contributors/workflow.md]: `hooks/pre-commit` is located in the tidb source code directory, not in `../../`

Comment (liuxintong) [reply contributors/workflow.md]: Yes, but the real path of `pre-commit` is `.git/hooks/../../hooks/pre-commit`. By the way, this is easy to test in the local environment.

Comment (zz-jason) [reply contributors/workflow.md]: gotcha

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**